core/drivers/bcm_gpio.c

```diff
@@ -0,0 +1,148 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright 2019 Broadcom.
+ */
+#include <assert.h>
+#include <gpio.h>
+#include <initcall.h>
+#include <io.h>
+#include <kernel/misc.h>
+#include <mm/core_memprot.h>
+#include <mm/core_mmu.h>
+#include <platform_config.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <string.h>
```

Comment:
needed?  (my comment covers **string.h**, **stdlib.h**, **stdint.h**, **misc.h**).
Also no need to include **core_mmu.h**, already included from **core_memprot.h**.

Reply:
thanks. will clean up.

Reply:
done